src/Common/src/Interop/Windows/winhttp/Interop.winhttp_types.cs

```diff
@@ -310,5 +310,13 @@ public struct WINHTTP_WEB_SOCKET_STATUS
             public uint dwBytesTransferred;
             public WINHTTP_WEB_SOCKET_BUFFER_TYPE eBufferType;
         }
+
+        [Flags]
```

Comment:
For flag bits like this in p/invoke's, we usually don't make them enum type.  We simply make them `public const uinit` and keep the native names.  

See for example: https://github.com/dotnet/corefx/blob/master/src/Common/src/Interop/Windows/winhttp/Interop.winhttp_types.cs#L108

So, this would probably be done as this:

```c#
public const uint WINHTTP_AUTO_DETECT_TYPE_DHCP  = 0x00000001;
public const uint WINHTTP_AUTO_DETECT_TYPE_DNS_A = 0x00000010;
```

And then you would reference it in code with `Interop.WinHttp.WINHTTP_AUTO_DETECT_TYPE_DHCP`, etc.

Reply:
Will change to follow the same pattern.